P/PGPLOT/build_tarballs.jl

```diff
@@ -5,30 +5,54 @@ using BinaryBuilder, Pkg
 name = "PGPLOT"
 version = v"5.2"
 
-sources = [ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4")]
+sources = [
+    ArchiveSource("ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz", "a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4"),
+    DirectorySource("./bundled"),
+]
 
 script = raw"""
 cd $WORKSPACE/srcdir
+
+if [[ "${target}" == *-apple-* ]]; then
+    cd pgplot
+    for f in ../patch-*;
+    do
+    patch -p0 <$f;
+    done
+    mkdir sys_darwin
+    cp ../bb.conf ./sys_darwin/bb.conf
+    cd ..
+fi
+
 mkdir pgplot_build && cd pgplot_build/
-cat ../pgplot/drivers.list | sed 's|! PSDRIV|  PSDRIV|g' | sed 's|! GIDRIV|  GIDRIV|g' > drivers.list
-../pgplot/makemake ../pgplot/ linux g77_gcc
-sed -i 's|FCOMPL=g77|FCOMPL=gfortran|' makefile
-make lib SHARED_LD="${FC} -shared  -o libpgplot.${dlext}"
-cp libpgplot.so "${libdir}/libpgplot.so"
+cat ../pgplot/drivers.list | sed 's|! PSDRIV|  PSDRIV|g' | sed 's|! GIDRIV|  GIDRIV|g' | sed 's|! PNG|  PNG|g' > drivers.list
+
+if [[ "${target}" == *-apple-* ]]; then
+    ../pgplot/makemake ../pgplot/ darwin
+    make lib SHELL=/bin/bash
+else
+    ../pgplot/makemake ../pgplot/ linux g77_gcc
+    sed -i 's|FCOMPL=g77|FCOMPL=gfortran|' makefile
+    make lib SHARED_LD="${FC} -shared  -o libpgplot.${dlext}"
+fi
+
+cp libpgplot.${dlext} "${libdir}/libpgplot.${dlext}"
```

Comment:
Just to make the command more verbose
```suggestion
install -Dvm 755 libpgplot.${dlext} "${libdir}/libpgplot.${dlext}"
```